### DISCUSSION TITLE
How to unobserve

### DISCUSSION THREAD

**Discussion by bjrnschmdt** [thebuilder/react-intersection-observer#504] [Q&A] [answered]
Hi there,

I wonder if someone could give me a hand.

I created a hook like this:
`const { ref, inView, entry } = useInView(options);`

I want to unobserve the referenced element at some point. How would I do that? I found this but I don't know how to use it:
`import { observe } from 'react-intersection-observer';`
`const destroy = observe(element, callback, options);`

Mainly I don't know how to populate the arguments. 
Do I have to use getElementById() to get the element?
`element = Document.getElementById()`
Or is it somehow possible to use the `ref` from the hook?
A callback is required but I have no clue what callback I should insert.
Are the options the same options as in the useInView hook?

And finally how do I unobserve the element? Just by calling `destroy()`?

Sorry, I know it is a total newbie question but I am grateful for all the help I can get, thanks!

**Comment by thebuilder** [comment, marked as answer]
You can pass in `skip` in the options to unobserve.

```js
const { ref, inView, entry } = useInView({
  skip: true // or false
});
```

Don't use `observe` directly unless you have a good reason not to use the hook. :)

**Reply by bjrnschmdt** [reply]
It works like charm! Thank you very much! I learned a great deal from your answer! :)
